src/app/@ansyn/ansyn/services/overlay-out-of-bounds/overlay-out-of-bounds.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable, OnDestroy, OnInit } from '@angular/core';
+import { take } from "rxjs/operators";
+import { Store } from "@ngrx/store";
+import { selectActiveMapId } from "@ansyn/map-facade";
+import { bboxFromGeoJson, ImageryCommunicatorService } from "@ansyn/imagery";
+
+
+@Injectable({
+	providedIn: 'root'
+})
+export class OverlayOutOfBoundsService implements OnInit, OnDestroy {
+
+	constructor(protected store$: Store, protected communicatorService: ImageryCommunicatorService) {
+	}
+
+	ngOnInit(): void {
+	}
+
+	ngOnDestroy(): void {
+	}
+
+	backToExtent(): void {
```

Comment:
This new service is only for one function? 
Maybe we can add the function to one of the existing services instead?